src/log/log.c

```diff
@@ -751,7 +753,6 @@ __wt_log_fill(
         WT_ERR(__log_fs_write(session, myslot->slot,
           myslot->offset + myslot->slot->slot_start_offset, record->size, record->mem));
 
-    WT_STAT_CONN_INCRV(session, log_bytes_written, record->size);
```

Comment:
The meaning of this statistic is number of bytes written into the log, not specifically the bytes written to the disk at this moment in time. Capturing the bytes as the threads write into the shared memory captures each thread's write rather than waiting until the buffer is eventually written to disk.

Reply:
@sueloverso 
Thank you for your reply, which makes me understand the logic here more deeply.
But the record with the WT_MYSLOT_UNBUFFERED identifier is not written to shared memory, but directly to disk.
Do we need to split log_bytes_written into two new statistics? For example, log_bytes_written_buffered + log_bytes_written_unbuffered? If necessary I will modify here, if not I will close this PR.

thanks.
:)

Reply:
The statistic is to know how many bytes are written to the logs in any form. The buffered/unbuffered is just a mechanism based on the memory size. We've never had a need to split these out. I don't think we need this PR.

Reply:
got it, thanks.
I close this PR.